src/api/spec/controllers/worker/capability_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Worker::CapabilityController, vcr: true, type: :controller do
+  render_views
+
+  let(:user) { create(:confirmed_user) }
+
+  before do
+    login user
+  end
+
+  describe 'GET /show' do
+    context 'worker exists' do
+      subject! { get :show, params: { worker: 'x86_64:7de14aea3d12:1' } }
```

Comment:
this sounds pretty specific - is this yet another cassette trick?

Reply:
i think, if you record the cassette looking for a specific value from backend, then you should lookup for this value... no?

Reply:
This means you can't rerun your test without your cassette, yes. Because everytime the worker docker container will have a different ID

Reply:
which doesn't matter because I'm not testing the backend but my controller, so if we wanted to test the backend together then you are right!

Reply:
sure - but don't use a cassette trick for that - use a specific mock response

Reply:
I agree with @coolo and it does matter because when you change a test and need to record the cassette again, this will fail and one has to debug / change the code.

Reply:
> Nope! If you rerun the test without cassettes it should also succeed otherwise it does not make sense to test it, right?

well it succeeds! The question is if it will succeed in other machines - it works by accident - as many other corners in our development environment ;) but I will update it

Reply:
It does not as soon as you rebuild your docker containers!

> many other corners in our development environment 

That is not true, most tests work if you regenerate the cassettes (with other containers and on other machines). If there are single tests which do not work, then it is a bug. And @coolo worked in the past to remove these 'flaky' tests (see https://github.com/openSUSE/open-build-service/pull/5308). And just because there are some tests which are failing does not mean we should introduce more flaky tests IMHO.

Reply:
yup, but note that i wrote "development environment" not 'flaky tests' ;)

Reply:
[nitpick] I would prefer sth more human readable instead of a hash, e.g. ``hostname``